Functions/xASL_bids_MergeNifti_RenameParms.m

```diff
@@ -0,0 +1,31 @@
+function xASL_bids_MergeNifti_RenameParms(Fpath,Fname)
```

Comment:
@jan-petr This looks like an admin function that we don't need right, _parms is legacy?